company.el

```diff
@@ -2321,16 +2334,17 @@ character, stripping the modifiers.  That character must be a digit."
   (interactive)
   (let (other-window-scroll-buffer)
     (company--electric-do
-      (let* ((selected (nth company-selection company-candidates))
-             (doc-buffer (or (company-call-backend 'doc-buffer selected)
-                             (user-error "No documentation available")))
-             start)
-        (when (consp doc-buffer)
-          (setq start (cdr doc-buffer)
-                doc-buffer (car doc-buffer)))
-        (setq other-window-scroll-buffer (get-buffer doc-buffer))
-        (let ((win (display-buffer doc-buffer t)))
-          (set-window-start win (if start start (point-min))))))))
+      (when company-selection
```

Comment:
Nitpick, feel free to disregard:
Since calling this function shouldn't do anything without a selection, I suggest moving the check for `company-selection` to the beginning of the function. I think that generally code that exits as early as possible is easier to reason about, so instead of aborting inside `electric-do` it's preferable to abort before.